client/src/Pages/ForgetPassword.jsx

```diff
@@ -1,28 +1,53 @@
 import React, { useState } from "react";
 import { Link } from "react-router-dom";
 import { Typography, Snackbar, Alert } from "@mui/material";
+import axios from "axios";
 import ForgetPasswordCom from "../Components/LoginPage/ForgetPassword";
 
 export default function ForgetPassword() {
+  const [loading, setLoading] = useState(false);
   const [alertMessage, setAlertMessage] = useState({
     message: "",
     open: false,
     vertical: "top",
     horizontal: "center",
+    severity: "",
```

Comment:
```suggestion
    severity: "success",
```
as the empty value warn an error at the console